<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="to_main_window_button">
     <property name="text">
      <string>На главную</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="table"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
